src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/transpose.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <tuple>
+#include <string>
+#include <vector>
+#include <memory>
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+namespace ov {
+namespace test {
+typedef std::tuple<
+        std::vector<size_t>,            // Input order
+        ov::element::Type,              // Model type
```

Comment:
Minor: `Model type` -> `Input type`

Reply:
Input/Output types are reserved for `inType`/`outType` test class members to be applied via `PrePostProcess`